app/models/decidim/challenges/challenge.rb

```diff
@@ -3,7 +3,7 @@
 module Decidim
   module Challenges
     # The data store for a Challenge in the Decidim::Challenges component.
-    class Challenge < Challenges::ApplicationRecord
+    class Challenge < Decidim::ApplicationRecord
```

Comment:
Here you should do a merge with master, because this has changed. All models extend from its own application record.
In this case, this should be `class Challenge < Challenges::ApplicationRecord`
- https://github.com/gencat/decidim-challenges/blob/3516a7c4c83e13e49e794e0278a994bd6496fe23/app/models/decidim/challenges/challenge.rb#L6